### PR TITLE
MEN-4335

### PR DESCRIPTION
**Pull request by mzedel**
the relevant change is in https://github.com/mendersoftware/gui/pull/1296/commits/57e2a98c5f8545e0df8b069aae6bcb200291ebb2

### CODE REVIEW

File: src/js/helpers.js [hunk 1]
```diff
@@ -494,7 +498,7 @@ systemctl restart mender-client && \\
   "ShellCommand": "/bin/bash"
 }
 EOF
-) && systemctl restart mender-connect'
+) && systemctl restart mender-connect${setupConfirmation}'
```
Comment (lluiscampos) [inline src/js/helpers.js:501]: This is a bit cheating: the "Running mender setup for hosted.mender.io" message should be printed *before* calling `mender-setup`, line 491.

Also, the "hosted.mender.io" part was meant to be an example, if do onboarding with a demo om-prem installation I expect this to print something else. Maybe is better to just say "Running mender setup"

**Review by lluiscampos — CHANGES_REQUESTED** [review]

File: src/js/helpers.js (hunk 1 above)
Comment (mzedel) [reply src/js/helpers.js]: ah right, I guess I misunderstood the reasoning behind the call... should now be more in line with the intention (I hope)

**Review by lluiscampos — APPROVED** [review]
Now it looks good! Thanks and sorry for the bad phrasing in the JIRA task description

**Comment by mzedel** [comment]
Ah I should have just looked better and switched my brain on a bit 🙈

**Pull request merged**